### PR TITLE
modified Makefile issue #120 on OS X

### PR DESCRIPTION
**Pull request by bryanhanson**
This is a simple change to accommodate the fact that Mac OS X does not have `find -daytime`.  I think the new incantation will work on any platform, and will cause `DESCRIPTION` to be updated (by modifying the version number to include the current date) if it is > 24 hrs old.  I hope.

I'm asking Erick (on Mac) & Roman (Ubuntu) to check.

Looks like there is a 2nd file in this commit for reasons I don't understand: https://github.com/cbeleites/hyperSpec/compare/develop...issue/120-daytime?expand=1#diff-95065ba29e79be839ee5ea1575847a70 This is not shown in my `git status` prior to push, and it's probably created by the build process and irrelevant here.  I don't think it should be merged, not sure how to disable that at this point however.

### CODE REVIEW

File: Vignettes/fileio/txt.Shimadzu/UV-VIS-NIR.TXT [hunk 1]
```diff
@@ -1,3 +1,1154 @@
-version https://git-lfs.github.com/spec/v1
-oid sha256:baabcf0c9248f75c3f48fbf6640cefdb172a5558e47cc9bf72e847ef994df334
-size 16956
+[SUMMARY]
```
Comment (ximeg) [inline Vignettes/fileio/txt.Shimadzu/UV-VIS-NIR.TXT:1]: Was it intended to have this file here? Apparently you included it by mistake

File: Makefile [hunk 2]
```diff
@@ -53,7 +53,7 @@ roxygenize: DESCRIPTION hyperSpec/R/*.R
 	@echo "Roxygenize"
 	@Rscript --vanilla -e "library (methods, quietly=TRUE, verbose = FALSE); library (devtools, quietly=TRUE, verbose = FALSE); document ('hyperSpec')"
 
-DESCRIPTION: $(shell find hyperSpec -maxdepth 1 -daystart -not -ctime 0 -name "DESCRIPTION") #only if not modified today
+DESCRIPTION: $(shell find hyperSpec -maxdepth 1 -not -ctime 1 -name "DESCRIPTION") #only if not modified in the past 24 hrs
```
Comment (ximeg) [inline Makefile:56]: It works under Ubuntu without any issues!

File: Vignettes/fileio/txt.Shimadzu/UV-VIS-NIR.TXT (hunk 1 above)
Comment (bryanhanson) [reply Vignettes/fileio/txt.Shimadzu/UV-VIS-NIR.TXT:1]: No.  New commit created which I think has only `Makefile`.

File: Makefile (hunk 2 above)
Comment (eoduniyi) [reply Makefile]: Still having trouble...
Screen Shot 2020-05-12 at 4 20 23 PM

Comment (ximeg) [reply Makefile]: @eoduniyi what command did you run? `make bootstrap` or `make`? Did you get all large files with `git lfs pull` before?

File: Vignettes/fileio/txt.Shimadzu/UV-VIS-NIR.TXT (hunk 1 above)
Comment (ximeg) [reply Vignettes/fileio/txt.Shimadzu/UV-VIS-NIR.TXT]: I reverted the accidentally changed file. Here is how you do it:
```
$ git checkout issue/120-daytime 
Switched to branch 'issue/120-daytime'
Your branch is up to date with 'origin/issue/120-daytime'.

$ git pull
Already up to date.

#######################################
# Restore the last version from `develop` branch
#######################################
$ git checkout develop -- Vignettes/fileio/txt.Shimadzu/UV-VIS-NIR.TXT

$ git add Vignettes/fileio/txt.Shimadzu/UV-VIS-NIR.TXT

#######################################
# Display all the changes that this branch introduces
#######################################
$ git diff develop
diff --git a/Makefile b/Makefile
index 6d66ca1..cc120c0 100644
--- a/Makefile
+++ b/Makefile
@@ -53,7 +53,7 @@ roxygenize: DESCRIPTION hyperSpec/R/*.R
        @echo "Roxygenize"
        @Rscript --vanilla -e "library (methods, quietly=TRUE, verbose = FALSE); library (devtools, quietly=TRUE, verbose = FALSE); document ('hyperSpec')"
 
-DESCRIPTION: $(shell find hyperSpec -maxdepth 1 -daystart -not -ctime 0 -name "DESCRIPTION") #only if not modified today
+DESCRIPTION: $(shell find hyperSpec -maxdepth 1 -not -ctime 1 -name "DESCRIPTION") # only if not modified in the past 24 hrs
        @echo update DESCRIPTION
        @sed "s/\(^Version: .*-\)\(20\)\?[0-9][0-9][0-1][0-9][0-3][0-9]\(.*\)$$/\1$(DATE)\3/" hyperSpec/DESCRIPTION > .DESCRIPTION
        @sed "s/\(^Date: .*\)20[0-9][0-9]-[0-1][0-9]-[0-3][0-9]\(.*\)$$/\1`date +%F`\2/" .DESCRIPTION > hyperSpec/DESCRIPTION

$ git commit

$ git push 
```

**Comment by bryanhanson** [comment]
The problem Erick sees is #111 and I still see it — my change to the Makefile was not intended to fix this, I believe only Claudia knows how to track this one down.

**Comment by eoduniyi** [comment]
@ximeg :
Screen Shot 2020-05-12 at 4 43 00 PM

File: README.md [hunk 3]
```diff
@@ -18,10 +18,16 @@ open `R` and use command `install.packages("hyperSpec")`. This installs the late
...
-1. Clone or download [hyperSpec source code from github](https://github.com/cbeleites/hyperSpec)  
+   **NOTE:** Windows users need to download and install Rtools(see this [instructions](https://cran.r-project.org/bin/windows/Rtools/)). 
+   ***`GNU make` is included to Rtools, so you don't need to install it.*** 
+   See Troubleshooting below to check version of `GNU make` and update it if needed. 
+   
+1. Clone [hyperSpec source code from github](https://github.com/cbeleites/hyperSpec)  
    `git clone git@github.com:cbeleites/hyperSpec.git`
 
+   **NOTE:** the hyperSpec repository uses `git lfs` for more efficient storage and handling of our collection of spectra files in various formats. 
+   *Downloading* a zip of the repository will leave you with so-called [pointer files]() only. These are not sufficient to build **hyperSpec**.
```
Comment (ximeg) [inline README.md:29]: I like this comment

**Pull request merged**